react-native/android/app/react.gradle

```diff
@@ -0,0 +1,75 @@
+def config = project.hasProperty("react") ? project.react : [];
```

Comment:
Due to changes in the assets system both ios and android now build the bundle as part of their build process. Hence I've removed the commands from npm